test/transcoding/FPGAIVDepLoopAttr.ll

```diff
@@ -52,7 +52,7 @@
 ; }
 
 ; RUN: llvm-as < %s > %t.bc
-; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_fpga_loop_controls -o %t.spv
```

Comment:
May we add test without specifying this extension to check that intel loop controls won't be translated and Extension/Capability won't be added in SPIR-V?

Reply:
That make sense, thanks!